ts/node/sql.ts

```diff
@@ -3244,15 +3400,17 @@ function removeAllClosedGroupEncryptionKeyPairs(groupPublicKey: string) {
  */
 function getAllV2OpenGroupRooms() {
   const rows = assertGlobalInstance()
-    .prepare(`SELECT json FROM ${OPEN_GROUP_ROOMS_V2_TABLE};`)
+    .prepare(`SELECT * FROM ${OPEN_GROUP_ROOMS_V2_TABLE};`)
     .all();
 
-  return map(rows, row => jsonToObject(row.json));
+  return rows;
 }
 
 function getV2OpenGroupRoom(conversationId: string) {
   const row = assertGlobalInstance()
-    .prepare(`SELECT * FROM ${OPEN_GROUP_ROOMS_V2_TABLE} WHERE conversationId = $conversationId;`)
+    .prepare(
+      `SELECT json FROM ${OPEN_GROUP_ROOMS_V2_TABLE} WHERE conversationId = $conversationId;`
+    )
```

Comment:
are open group rooms not converted into columns yet?

Reply:
only the conversation table is being migrated right now to columns, so yes, they opengroups room table are not migrated yet

Reply:
👍